### DISCUSSION TITLE
Installation Issue on Azure cloud POC environment

### DISCUSSION THREAD

**Discussion by manish050994** [Sunbird-cQube/community#811] [Q&A]
Script which we are using is not compliance as per POC environment, This script auto provided IP address in 10.X.X.X series and public IP which is not compatible .
Server is crashing during the installation on Azure Cloud POC environment.

**Comment by DevashishRaghav** [comment]
@asifbasha61

**Comment by RNarmatha** [comment]
Hi Manish - Asif replied for this issue  in the whats app group

**Comment by rhwarrier** [comment]
@RNarmatha @asifbasha61  Is this issue resolved ?

**Comment by RNarmatha** [comment]
Hi Rahul, today Asif will work on it.

@Abdul Asif ***@***.***> - please update the status

**Comment by RNarmatha** [comment]
@asifbasha61 - Please update the status on Azure
